docs/additional-functionality/rapids-shuffle.md

```diff
@@ -77,6 +77,44 @@ The minimum UCX requirement for the RAPIDS Shuffle Manager is
    
    ---
    
+##### CentOS UCX RPM
+The UCX packages for CentOS 7 and 8 are divided into different RPMs. For example, for UCX 1.10.1, 
+fetched from https://github.com/openucx/ucx/releases/download/v1.10.1/ucx-v1.10.1-centos7-mofed5.x-cuda11.0.tar.bz2, 
+is a tarball that contains:
+
+```
+ucx-devel-1.10.1-1.el7.x86_64.rpm   
+ucx-debuginfo-1.10.1-1.el7.x86_64.rpm
+ucx-1.10.1-1.el7.x86_64.rpm         
+ucx-cuda-1.10.1-1.el7.x86_64.rpm
+ucx-rdmacm-1.10.1-1.el7.x86_64.rpm  
+ucx-cma-1.10.1-1.el7.x86_64.rpm
+ucx-ib-1.10.1-1.el7.x86_64.rpm
+```
+
+For a setup without RoCE or Infiniband networking, the only packages required are: 
+
+```
+ucx-1.10.1-1.el7.x86_64.rpm, and 
+ucx-cuda-1.10.1-1.el7.x86_64.rpm. 
+```
+
+If the accelerated networking is available, the package list is: 
+
+```
+ucx-1.10.1-1.el7.x86_64.rpm, 
+ucx-cuda-1.10.1-1.el7.x86_64.rpm,
+ucx-rdmacm-1.10.1-1.el7.x86_64.rpm, and
+ucx-ib-1.10.1-1.el7.x86_64.rpm.
+```
+
+---
+**NOTE:**
+
+The CentOS RPM requires CUDA installed via RPMs to satisfy its dependencies.
```

Comment:
Is there a useful link we can place here to direct users on how to install CUDA via RPM?  Or maybe point to the CUDA toolkit front-page and walk them through how to navigate that site to instructions on installing via RPM?

Reply:
Sure thing, will add.

Reply:
added a specific example here: https://github.com/NVIDIA/spark-rapids/pull/2537/commits/d132c7f1730a0fb9a293dbd436b3afd7a304e17a, though its debatable whether it "fits" the UCX doc.